### PR TITLE
Use batch delete endpoint

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

File: verification/curator-service/api/src/controllers/cases.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ export default class CasesController {
         try {
             const response = await axios.delete(
                 this.dataServerURL + '/api' + req.url,
-                req.body,
+                { data: req.body },
```
Comment (attwad) [inline verification/curator-service/api/src/controllers/cases.ts:106]: why do we need the data: indirection here while all the other methods proxying just pass the req.body as is ?

Comment (attwad) [inline verification/curator-service/api/src/controllers/cases.ts:106]: also if really needed, should data be mentioned in the openapi spec ?

**Review by attwad — APPROVED** [review]

File: verification/curator-service/api/src/controllers/cases.ts (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/api/src/controllers/cases.ts:106]: Axios quirk 🤷‍♀️   took me awhile to figure it out https://github.com/axios/axios/issues/736

Comment (allysonjp715) [reply verification/curator-service/api/src/controllers/cases.ts:106]: No, in the final request it is sent as the actual body. Just a weird thing with axios.

**Pull request merged**